manifests.go

```diff
@@ -0,0 +1,43 @@
+package distribution
+
+import "github.com/docker/distribution/digest"
+
+// Manifest specifies a registry object with a list of dependencies.
+type Manifest interface {
+	Blob // io access to serialized content.
+
+	// Dependencies returns a list of object on which this manifest depends.
+	// Typically, these are layers.
+	Dependencies() []Descriptor
```

Comment:
It's part of the blob. It's up to the application to abstract it but it may be deserialized as part of the ManifestService.